recipes/_user_group.rb

```diff
@@ -0,0 +1,23 @@
+# setup kafka group
+group node["kafka"]["group"] do
+  action :create
+end
+
+# setup kafka user
+user node["kafka"]["user"] do
+  comment "Kafka user"
+  uid node["kafka"]["uid"] if node["kafka"]["uid"]
+  gid node["kafka"]["group"]
+  shell "/bin/bash"
+  home "/home/#{node["kafka"]["user"]}"
+  supports :manage_home => true
+end
+
+# Configure kafka user
+template "/home/#{node["kafka"]["user"]}/.bash_profile" do
+  source  "bash_profile.erb"
+  owner node["kafka"]["user"]
+  group node["kafka"]["group"]
+  mode  00755
+  notifies :restart, "service[kafka]"
```

Comment:
I shouldn't have included this `template` block in `_user_group.rb`...

```
ERROR: resource template[/home/kafka/.bash_profile] is configured to notify resource service[kafka] with action restart, but service[kafka] cannot be found in the resource collection. template[/home/kafka/.bash_profile] is defined in /tmp/kitchen/cookbooks/cerner_kafka/recipes/_user_group.rb:17:in `from_file'
```

Right now when I add `recipe[cerner_kafka::_user_group]` to my run list before `recipe[cerner_kafka]`, chef-client:

1. creates the group
1. creates the user
1. templates the user's .bash_profile
  1. tries to restart `service[kafka]` which chef-client doesn't know exists

I'll be submitting a pr shortly to move the `template .bash_profile` block back into the default recipe

@bbaugher @mkwhitacre